frontend/components/Chat.tsx

```diff
@@ -0,0 +1,246 @@
+  const handleClickSend = async () => {
+    const keyword = textarea;
+    setTextarea("");
+    pushChat({ type: "user", body: <p>{keyword}</p> });
+    await sleep(1000);
+    pushChat({
+      type: "bot",
+      body: <p>「{keyword}」ですね！探してみます 🚲</p>,
+    });
+    pushChat({
+      type: "bot",
+      body: <Wait />,
+    });
+    await sleep(1000);
+    try {
+      const { badges: badgesList, total_count } =
+        await client.wisdomBadges.list.keyword.$get({
+          query: { keyword },
+        });
+      popChat();
+      total_count === 0
+        ? pushChat({
+            type: "bot",
+            body: (
+              <p>
+                すみません、「{keyword}」
+                に関する能力バッジがみつかりませんでした。。
+              </p>
+            ),
+          })
+        : pushChat({
+            type: "bot",
+            body: (
+              <>
+                <p className="mb-4">
+                  「{textarea}」に関する能力バッジが{total_count}
+                  件見つかりました。
+                </p>
+                <ul className="pl-8 list-disc mb-4">
+                  {badgesList.slice(0, 3).map((badges) => (
+                    <li key={badges.badges_id}>
+                      <Link
+                        href={pagesPath.wisdom_badges
+                          ._wisdomBadgesId(badges.badges_id)
+                          .$url()}
+                      >
+                        <a className="underline text-primary-700">
+                          {badges.name}
+                        </a>
+                      </Link>
+                    </li>
+                  ))}
+                </ul>
+                <p>
+                  <Link
+                    href={pagesPath.search.$url({ query: { q: textarea } })}
+                  >
+                    <a className="text-primary-700">
+                      …他{total_count}件の検索結果を確認する
+                    </a>
+                  </Link>
+                </p>
+              </>
+            ),
+          });
+    } catch (_) {
+      popChat();
+      pushChat({
+        type: "bot",
+        body: (
+          <p>
+            すみません、探している間に問題が発生したようです。。しばらくしてからもう一度お試しください。
+          </p>
+        ),
+      });
+    }
+  };
+  return (
+    <div className={clsx("", className)}>
+      <button
+        className="absolute bottom-0 right-0 p-4 rounded-full bg-primary-700 shadow-xl hover:bg-primary-600 mb-8"
+        aria-controls={id}
+        title="ボットと会話する"
+        onClick={handleClickExpand}
+      >
+        <Icon className="text-white text-3xl" icon="fa-regular:comment-dots" />
+      </button>
+      <nav
+        id={id}
+        className={clsx(
+          "absolute w-[24rem] bottom-0 right-0 bg-white shadow-xl transition-transform duration-150 ease-in-out",
+          {
+            ["-bottom-full translate-y-full"]: !expand,
+          }
+        )}
+      >
+        <button
+          className="w-full flex items-center bg-primary-700 rounded-t-xl p-4 text-white hover:bg-primary-600"
```

Comment:
```suggestion
          className="w-full flex items-center bg-primary-700 rounded-t-xl p-4 text-white hover:bg-primary-600"
          aria-label="閉じる"
```

閉じるためのボタンであることをスクリーンリーダーが読み上げられたほうがアクセシビリティの観点でよりよい